cypress/support/fragments/orders/orderLineEditForm.js

```diff
@@ -28,6 +28,11 @@ const cancelButton = Button('Cancel');
 const saveButton = Button('Save & close');
 const saveAndOpenOrderButton = Button('Save & open order');
 
+const title = TextField({ name: 'titleOrPackage' });
```

Comment:
Constant for the same element is already initialized in itemDetailsFields object